mgmt/rest/client/client.go

```diff
@@ -118,7 +124,7 @@ func (c *Client) do(method, path string, ct contentType, body ...[]byte) (*rbody
 			if strings.Contains(err.Error(), "tls: oversized record") || strings.Contains(err.Error(), "malformed HTTP response") {
 				return nil, fmt.Errorf("error connecting to API URI: %s. Do you have an http/https mismatch?", c.URL)
 			}
-			return nil, err
+			return nil, fmt.Errorf("URL target is not available. %v", err)
```

Comment:
There are two changes in this PR. One that addresses issue #260. A second change that changes error messages returned after a request is made to the API.